api/v2/controllers/bouquets/bouquets_controller.py

```diff
@@ -0,0 +1,34 @@
+from flask import make_response, jsonify, request
+from flask_restful import Resource
+
+from api.v2.helpers.bouquets.bouquets_helper import query_all_bouquets
+from api.v2.models.bouquets.bouquets_model import Bouquets as BouquetsModel
+from api.v2.utilities.validators import validate_bouquet_adding
+from api.v2.helpers.credentials import calendar
+
+class Bouquets(Resource):
+    method_decorators = [validate_bouquet_adding]
+
+    def get(self):
+        bouquets = query_all_bouquets()
+        return make_response(jsonify({"bouquets": bouquets}), 200)
+
+    def post(self):
+        data = request.get_json()
+        redirect_uris = ''
+        if 'should_refresh' not in data.keys():
+            data['should_refresh'] = False
+        if 'refresh_url' not in data.keys():
+            data['refresh_url'] = 'http://www.google.com'
+        for i in data['redirect_uris']:
+            redirect_uris += f'{i} '
+        data['redirect_uris'] = redirect_uris
+        result = calendar(data)
```

Comment:
This function `calendar` is being called in both the validator and in the `post` method. Aren't you doing double work?